internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -2325,3 +2346,27 @@ func getClusterDeploymentAdminKubeConfigSecretName(cd *hivev1.ClusterDeployment)
 	}
 	return fmt.Sprintf(adminKubeConfigStringTemplate, cd.Name)
 }
+
+// processMirrorRegistryConfig retrieves the mirror registry configuration from the referenced ConfigMap
+func (r *ClusterDeploymentsReconciler) processMirrorRegistryConfig(ctx context.Context, log logrus.FieldLogger, clusterInstall *hiveext.AgentClusterInstall) (*hiveext.MirrorRegistryConfiguration, error) {
```

Comment:
We should label this configmap with the backup label.

I think there's a function already for that.